blog/src/User/Console/CreateCommand.php

```diff
@@ -41,25 +46,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $login = (string) $input->getArgument('login');
         $password = (string) $input->getArgument('password');
         $isAdmin = (bool) $input->getArgument('isAdmin');
-
-        $this->signupForm->load([
-            'login' => $login,
-            'password' => $password,
-            'passwordVerify' => $password,
-        ], '');
-
         try {
+            $this->formHydrator->populate(model: $this->signupForm, data: [
+                'login' => $login,
+                'password' => $password,
+                'passwordVerify' => $password,
+            ], scope: '');
             $user = $this->signupForm->signup();
         } catch (Throwable $t) {
             $io->error($t->getMessage() . ' ' . $t->getFile() . ' ' . $t->getLine());
-
             return $t->getCode() ?: ExitCode::UNSPECIFIED_ERROR;
         }
 
         if ($user === false) {
```

Comment:
Psalm Level 1: Testing
ERROR: TypeDoesNotContainType - src/User/Console/CreateCommand.php:78:13 - Type App\User\User for $user is never falsy (see https://psalm.dev/056)
        if (null==$user) {

Reply:
Recommended: if (!$user instanceof User) {